funnel/utils/markdown/base.py

```diff
@@ -280,6 +287,7 @@ def render(self, text: Optional[str]) -> Optional[Markup]:
         'footnote',
         'footnote_ext',  # Must be after 'footnote' to take effect
         'heading_anchors',
+        'anchors_ext',  # Must be after 'heading_anchors' to take effect
```

Comment:
Should we call it `heading_anchors_fix`?

Reply:
Yes, sure!
Was just following the same convention as `footnotes_ext`.